### PR TITLE
Update BFE product name.

### PR DESCRIPTION
**Pull request by mohsen-bfe**

### CODE REVIEW

File: docs/5.0. NMOS Solutions.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Many of these products have participated in the JT-NM Tested programme, with res
 
 | Company | Product | Supported Specifications | Comments |
 | ------- | ------- | ------------------------ | -------- |
-| BFE | [SILKNET](https://bfe.tv/en/solutions-products/control-monitoring-systems/ksc-silknet/) | IS-04 v1.3 <br/> IS-05 v1.1 <br/> IS-07 v1.0  | Broadcast SDN Controller |
+| BFE | [KSC CORE,](https://www.bfe.tv/en/solutions-products/control-monitoring-systems/ksc-core/) <br/> [KSC SILKNET](https://bfe.tv/en/solutions-products/control-monitoring-systems/ksc-silknet/) | IS-04 v1.3 <br/> IS-05 v1.1 <br/> IS-07 v1.0  | Broadcast Controller, <br/> Broadcast SDN Controller |
```
Comment (garethsb) [inline docs/5.0. NMOS Solutions.md:43]:
```suggestion
| BFE | [KSC CORE] (https://www.bfe.tv/en/solutions-products/control-monitoring-systems/ksc-core/), <br/> [KSC SILKNET](https://bfe.tv/en/solutions-products/control-monitoring-systems/ksc-silknet/) | IS-04 v1.3 <br/> IS-05 v1.1 <br/> IS-07 v1.0  | Broadcast Controller, <br/> Broadcast SDN Controller |
```

**Review by garethsb — COMMENTED** [review]
Just want to confirm: the web page for KSC CORE only mentions IS-07, is that right? (I think the brochure doesn't mention NMOS at all.) The web page for KSC SILKNET mentions IS-04, IS-05 and IS-07, though the brochure doesn't mention IS-07, as far as I can see.

**Comment by mohsen-bfe** [comment]
@garethsb Thanks for your prompt response.

KSC CORE only mentions IS-07, because the IS-07 functionality is focused here (IS-04, and IS-05 are included implicitly)

I will update brochures as well.

**Comment by garethsb** [comment]
Thanks, makes sense to me. I expect @peterbrightwell will then approve / merge. 👍

**Comment by peterbrightwell** [comment]
Thanks, yes I'll approve and merge when updated.

P

**Comment by peterbrightwell** [comment]
@mohsen-bfe are updates done yet?

**Comment by mohsen-bfe** [comment]
@peterbrightwell Sorry for my delayed response, not yet, I will write here when are done. Thanks.

**Comment by peterbrightwell** [comment]
@mohsen-bfe any update on this?  The PR will now need conflicts resolved also.

**Review by garethsb — APPROVED** [review]
LGTM

**Comment by peterbrightwell** [comment]
THanks @mohsen-bfe , merging now

**Pull request merged**